.circleci/config.yml

```diff
@@ -98,9 +98,49 @@ jobs:
           command: bundle exec danger || echo "danger failed"
           when: always # Run this even when tests fail
 
+  "Docs":
+    docker:
+      - image: circleci/ruby:2.3
+    steps:
+      - checkout
+      
+      - restore_cache:
+          key: v1-ubuntu-gems-{{ checksum "Gemfile" }}
+      - run:
+          name: Setup Build
+          command: |
+            bundle check --path .bundle || bundle install --jobs=4 --retry=3 --path .bundle
+      - save_cache:
+          key: v1-ubuntu-gems-{{ checksum "Gemfile" }}
+          paths:
+            - .bundle
+
+      - run: bundle exec fastlane validate_docs
+
+  "Sourcecode":
```

Comment:
This is for rubocop? Maybe replace with "Lint"

Reply:
Rubocop, `ensure_no_debug_code`, Shellcheck.

Reply:
Yeah, I think all of those qualify as linting 😄